### PR TITLE
Fixes responsive SLC docs

### PR DESCRIPTION
**Pull request by garanj**
#### WHAT

Fixes responsive example for ScalingLazyColumn

#### WHY

Example doesn't build when copy/pasted

#### HOW

Fixed and confirmed build and execute

#### Checklist :clipboard:
- [N/A] Add explicit visibility modifier and explicit return types for public declarations
- [N/A] Run spotless check
- [N/A] Run tests
- [N/A] Update metalava's signature text files

### CODE REVIEW

File: docs/compose-layout.md [hunk 1]
```diff
@@ -41,7 +41,10 @@ Scaffold(
             }
         }
         items(10) {
-            Chip("Item $it", onClick = {})
+            Chip(
```
Comment (yschimke) [inline docs/compose-layout.md:44]: You need the import from compose-material

**Review by yschimke — CHANGES_REQUESTED** [review]
show an import?

File: docs/compose-layout.md (hunk 1 above)
Comment (garanj) [reply docs/compose-layout.md:44]: Do you mean the original would work with an additional import? I have androidx.wear.compose:compose-material 1.3.0.

Maybe 

```
Chip(onClick = {}) {
    Text("Item $it")
}
```

is better?

Comment (yschimke) [reply docs/compose-layout.md:44]: See https://google.github.io/horologist/compose-material/#compose-material

Comment (garanj) [reply docs/compose-layout.md:44]: Got it!!

**Comment by garanj** [comment]
Updated - apologies, makes sense now!😀

File: docs/compose-layout.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ with `responsive()` will handle all the following:
 - Size side margins based on a percentage, adapting to different screen sizes.
 
 ```kotlin
+import com.google.android.horologist.compose.material.Chip
```
Comment (yschimke) [inline docs/compose-layout.md:20]: TIOLI: should we import * here to not fix on specific classes. You are missing a lot of other imports.

**Comment by yschimke** [comment]
Should we add a banner on top once showing imports come from compose-material?

**Comment by garanj** [comment]
Added a note!

**Pull request merged**